src/transaction.js

```diff
@@ -19,7 +19,7 @@ Transaction.SIGHASH_NONE = 0x02
 Transaction.SIGHASH_SINGLE = 0x03
 Transaction.SIGHASH_ANYONECANPAY = 0x80
 
-Transaction.fromBuffer = function (buffer, __disableAssert) {
+Transaction.fromBuffer = function (buffer, __disableExcess) {
```

Comment:
`__disableExcess`, if provides, disables the 'strictness' of `fromBuffer` from checking that the bytes read are equal to the bytes in `buffer`.